op-chain-ops/genesis/config.go

```diff
@@ -246,6 +246,16 @@ type DeployConfig struct {
 	// proofs instead of the older output oracle mechanism.
 	UseFaultProofs bool `json:"useFaultProofs"`
 
+	// DaChallengeWindow represents the block interval during which the availability of a data commitment can be challenged.
```

Comment:
nit: this could be updated to have `usePlasma` which could ultimately get routed thru to the rollup config to tell the `op-node` to be in plasma mode. thinking longer term about what tells a node whether its in plasma mode or not, we might want to put something in the system config? not sure just thinking out loud, the reason is you never want to turn a plasma node on accidentally as a rollup so what level of abstraction is best for that?

Reply:
added `UsePlasma` here